### ISSUE TITLE
Feature request: Ability to backup/export data

### ISSUE THREAD

**Issue by jfdid** [TomBursch/kitchenowl#59]
### What's the feature 🧐

Loving this project.
It would be great to have the ability to export your data(grocery items, categories, recipes, etc) for backup in case you're moving servers/etc.  Could be a button on the GUI(web browser to download), etc.
I'm hesitating on diving right into add all of our recipes for fear of losing all of that work at some point.
Is there currently a way to navigate the docker container to backup the data for future releases, or moving to another server, etc(apart from backing up the actual container)?
Thanks again for your hard work.

**Comment by TomBursch** [comment]
So you're right there is currently no way in the UI to export data, but for everyone else interested I will give two more technical ways of exporting or backing up your data.
### Export
There are endpoints in the API that allow exporting items and/or recipes to JSON. 
They can be found under `URL/api/export`, `URL/api/export/items`, `URL/api/export/recipes`
You need to be logged in or use a long lived token to access these endpoints.

### Backup
So either export them as described before and when needed import them using `URL/api/import` or probably easier is to directly backup the database.

All persistent data is stored in `/data` inside the container which following the default configuration is bound to docker volume called `kitchenowl_data`. Inside this volume you will find a file called `database.db` this stores all your user/items/recipes and so on and a folder called `upload` where all images are stored. If you backup these files you will always be able to restore the state of kitchenowl.

With docker you could also configure these relevant files to be stored somewhere else on you file system for easier access.
So in this configuration I removed the volume and instead stored the persistent files somewhere directly in my file system (`/path/to/folder`)
```
version: "3"
services:
    front:
        image: tombursch/kitchenowl-web:latest
        environment:
        - FRONT_URL=http://localhost # The url the instance will be access with
        ports:
        - "80:80"
        depends_on:
        - back
        networks:
        - default
    back:
        image: tombursch/kitchenowl:latest
        restart: unless-stopped
        networks:
        - default
        environment:
        - JWT_SECRET_KEY=PLEASE_CHANGE_ME
        - FRONT_URL=http://localhost
        volumes:
        - /path/to/folder:/data # change your path here

networks:
    default:
```

**Comment by jfdid** [comment]
Thanks Tom,
This worked.  I copied the database.db file from the docker container.  I tested it by importing to a new container and it looks good.
For those wondering, I used the following command:
sudo docker cp kitchenowl_back_1:data/database.db backup
(where kitchenowl_back_1 is the container name, data/database.db is the folder within the container and backup is the folder on the host machine).
Just reverse the command to restore a backup:
sudo docker cp backup/database.db kitchenowl_back_1:data/

This restores items, categories, recipes and users.

I tried via the API but got an error for authentication header and that was over my head.  
Thanks again.
